### ISSUE TITLE
data2schema and schema_from_dataframe docs confusion/missing

### ISSUE THREAD

**Issue by layik** [cmudig/draco2#785]
Hi there. 

I am cautious of wasting your time here but struggling to use the docs especially [this notebook](https://dig.cmu.edu/draco2/applications/design_space_exploration.html). 

In my installed version 0.0.9 there is no function named "schema_from_dataframe" despite [this doc entry](https://dig.cmu.edu/draco2/api/schema.html?highlight=schema_from_dataframe#draco.schema.schema_from_dataframe), and for data2schema, there is [no such function](https://dig.cmu.edu/draco2/search.html?q=data2schema#) definition in the docs (as of 8th Jan 2024 as I cannot see a version) but "data2schema" does appear as shown below in my installed 0.0.9 versio:

```sh
>>> import draco
>>> dir(draco)
['Dict', 'List', '__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', '__version__', 'asp2vl', 'data2schema', 'data_to_asp', 'helper', 'is_valid', 'js', 'json', 'pd', 'read_data_to_asp', 'run', 'run_clingo', 'schema2asp', 'vl2asp']
>>> draco.__version__
'0.0.9'
>>> draco.data2schema.__doc__
>>> draco.is_valid.__doc__
' Check a task.\n        Args:\n            draco_query: a list of facts\n        Returns:\n            whether the task is valid\n    '
```

In short, I am struggling to generate a schema based on the the notebook illustration and paper I have read (_hopefully_ carefully).

Hope this helps rather than wasting your time, please feel free to close if it is just a noob confusion.

I must end by saying thank you for the great work (and tool).
